apps/frontend-manage/src/pages/courses/[id].tsx

```diff
@@ -49,8 +52,39 @@ function CourseOverviewPage() {
     <Layout>
       <div className="w-full mb-4">
         <div className="flex flex-row items-center justify-between">
-          <H1>Kurs: {course.name}</H1>
-          <div className="italic">{course.numOfParticipants} Teilnehmende</div>
+          <H1>
+            Kurs: {course.name} (PIN:{' '}
+            {String(course.pinCode)
+              .match(/.{1,3}/g)
+              ?.join(' ')}
+            )
+          </H1>
+          <div className="flex flex-row items-center gap-4 mb-2">
+            <QRPopup
+              relativeLink={`/course/${course.id}/join?pin=${course.pinCode}`}
+              triggerText="Kurs beitreten"
+              className={{ modal: 'w-[40rem]' }}
+            >
+              <H2>Kurs beitreten</H2>
+              <Link
+                href={`https://pwa.klicker.uzh.ch/course/${course.id}/join?pin=${course.pinCode}`}
```

Comment:
TODO: dynamic base URL based on new env variable in .env.production (e.g., NEXT\_PUBLIC\_PWA\_URL=https://pwa.klicker.uzh.ch) and same for other environments

Reply:
I added this directly to this PR so it that it can be used from here onwards and is not hard-coded over and over again and left for the end before the release.